pytest_selenium/pytest_selenium.py

```diff
@@ -123,7 +123,6 @@ def driver_kwargs(request, capabilities, chrome_options, driver_args,
         host=pytestconfig.getoption('host'),
         port=pytestconfig.getoption('port'),
         request=request,
-        log_path=None,
```

Comment:
I think we should change this to `service_log_path` instead of removing it. Otherwise a `driver_log` of `None` would result in the default Firefox driver's log path of `geckodriver.log` to be used.